examples/android/binder/java/io/grpc/binder/cpp/example/native.cc

```diff
@@ -15,6 +15,8 @@
 #include <android/log.h>
 #include <jni.h>
 #include "src/core/ext/transport/binder/client/channel_create.h"
+#include "examples/protos/helloworld.pb.h"
```

Comment:
Sort include headers. Maybe we can add this folder to the targets list of `clang_format_all_the_things.sh`?